components/camel-stax/src/main/java/org/apache/camel/component/stax/StAXProcessor.java

```diff
@@ -49,25 +51,34 @@ public StAXProcessor(ContentHandler contentHandler) {
 
     @Override
     public void process(Exchange exchange) throws Exception {
-        InputSource is = exchange.getIn().getMandatoryBody(InputSource.class);
```

Comment:
This lines open a new FileInputStream, but is not needed, as the method reader.parse(is) ignore this parameter